fluentd/run.sh

```diff
@@ -56,8 +56,8 @@ if [ "${USE_MUX:-}" = "true" ] ; then
             cp -f $file $CFG_DIR/openshift
         fi
     done
-    rm -f configs.d/dynamic/input-docker-* configs.d/dynamic/input-syslog-*
-    if [ "${MUX_ALLOW_EXTERNAL-}" = "true" ] ; then
+    (cd $CFG_DIR; rm -f dynamic/input-docker-* dynamic/input-syslog-*)
```

Comment:
Of course, that's fine, too.  I tried to preserve the original code as much as possible...